cfgov/v1/util/util.py

```diff
@@ -34,3 +37,35 @@ def get_unique_id(prefix='', suffix=''):
         'one_required': 'Please enter at least one date.'
     }
 }
+
+
+# Orders by most to least common in the list
+def most_common(lst):
```

Comment:
Can you rename this function to include the type it checks for? I'm assuming it takes a list of strings? so `most_common_string` or flesh out the comment so it explains what exactly is going on.